src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/singlespaceseparator/InputSingleSpaceErrors.java

```diff
@@ -1,4 +1,4 @@
-package  com.puppycrawl.   tools.checkstyle.checks.whitespace;
+package  com.puppycrawl.   tools.checkstyle.checks.whitespace.singlespaceseparator;
```

Comment:
file was not renamed. It should be `InputSingleSpaceSeparatorErrors`.
Class name can't be changed as stated before.